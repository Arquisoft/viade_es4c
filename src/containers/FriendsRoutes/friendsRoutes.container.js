
import React from "react";
import {routeHelper} from "../../viade";
import {RouteList} from "../../components/";

const FriendsRoutesComponent = (props) => {
	const {webId}=props;
		return (
			<RouteList webId={webId} readRoutes={routeHelper.fetchUrlSharedWithMeRoutes}/>
		);
};

export default FriendsRoutesComponent;