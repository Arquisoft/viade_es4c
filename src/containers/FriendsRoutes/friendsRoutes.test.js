import React from 'react';
import { shallow } from 'enzyme';
import FriendsRoutes from "./friendsRoutes.container";

describe('FriendsRoutes', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<FriendsRoutes debug />);
  
    expect(component).toMatchSnapshot();
  });
});
