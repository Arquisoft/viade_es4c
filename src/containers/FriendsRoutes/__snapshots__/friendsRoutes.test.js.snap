// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendsRoutes should render correctly in "debug" mode 1`] = `
<div>
  <Row
    className="justify-content-md-center"
    noGutters={false}
  >
    <Col
      key="col-1"
      lg={1}
      md={12}
      sm={12}
      xl={1}
      xs={12}
    />
    <Col
      key="col-2"
      lg={10}
      md={12}
      sm={12}
      xl={10}
      xs={12}
    >
      <Row
        noGutters={false}
      />
    </Col>
    <Col
      key="col-3"
      lg={1}
      md={12}
      sm={12}
      xl={1}
      xs={12}
    />
  </Row>
</div>
`;
